### ISSUE TITLE
Removing -L/usr/lib64, --start-group, --end-group for OSX

### ISSUE THREAD

**Issue by chemecse** [premake/premake-core#548]
Received the following linker warnings when running `premake5 gmake && make all` for this [project](https://github.com/networkprotocol/libyojimbo):
```
ld: warning: directory not found for option '-L/usr/lib64'
ld: unknown option: --start-group
```
In a closed issue (linked below) we determined it was a premake5 issue happening for older versions of OSX.

Manually removing all instances of `-L/usr/lib64`, `--start-group`, and `--end-group` from the generated Makefiles removes the warnings/errors.

System specific information and further issue discussion can be found [here](https://github.com/networkprotocol/libyojimbo/issues/1).

The warning produced by linking with `-L/usr/lib64` on OSX has previously been declared an [issue](https://github.com/premake/premake-core/issues/473) as well.

**Comment by starkos** [comment]
What version of Premake are you using? Have you tried it with the latest master branch?

**Comment by chemecse** [comment]
From the link with system specific information:
`premake5 --version` outputs: `premake5 (Premake Build Script Generator) 5.0.0-alpha9`

I will attempt to use master later on today.

**Comment by chemecse** [comment]
Using the latest master branch (`premake5 (Premake Build Script Generator) 5.0.0-dev`) seems to have removed the errors regarding the usage of `--start-group` and `--end-group`, but maintains the warnings about not finding `lib64`.

**Comment by starkos** [comment]
I believe this will be fixed by PR #543.

**Comment by samsinsane** [comment]
Closing as this issue should be resolved in master due to #543.
